README_.md

```diff
@@ -0,0 +1,175 @@
+# 1 BladeDISC
+
+BladeDISC is an end-to-end **DynamIc Shape Compiler** project for machine learning
+workloads, which is one of the key components of the Alibaba's [PAI-Blade](https://www.aliyun.com/activity/bigdata/blade).
+BladeDISC provides general, transparent and ease of use performance optimization
+for TensorFlow/PyTorch workloads on GPGPU and CPU backends. It supports multiple
+and flexible deployment solutions, including both Pluggin Mode inside
```

Comment:
Hi, Is the "pluggin" here a typo, perhaps it should be plugin?

Reply:
Thanks, it's updated.